qt/aqt/studydeck.py

```diff
@@ -152,6 +157,8 @@ def accept(self) -> None:
             showInfo(tr.decks_please_select_something())
             return
         self.name = self.names[self.form.list.currentRow()]
+        if callable(self.callback):
```

Comment:
A minor point, but you've tested for truthiness above, so you could probably do so here to be consistent.

Reply:
Updated as suggested